wls-gui-wahllokalsystem/src/composables/tasks/taskListService.ts

```diff
@@ -1,90 +1,76 @@
+import type { ExtendedWahlMetaData } from "@/composables/tasks/ExtendedWahlMetaData.ts";
 import type { Task } from "@/types/tasks/Task.ts";
 
 import { storeToRefs } from "pinia";
+import { computed } from "vue";
 
-import { useInfomanagementStore } from "@/stores/infomanagementStore.ts";
-import { useKopfdatenStore } from "@/stores/kopfdatenStore.ts";
+import { useKonfigurationsparameterTaskFactory } from "@/composables/tasks/taskFactories/konfigurationsparameterTaskFactory.ts";
+import { useKopfdatenTaskFactory } from "@/composables/tasks/taskFactories/kopfdatenTaskFactory.ts";
+import { useUngueltigeWahlscheineTaskFactory } from "@/composables/tasks/taskFactories/ungueltigeWahlscheineTaskFactory.ts";
+import { useWahlvorstandTaskFactory } from "@/composables/tasks/taskFactories/wahlvorstandTaskFactory.ts";
 import { useUserStore } from "@/stores/userStore.ts";
-import { useWahlbezirkStore } from "@/stores/wahlbezirkStore.ts";
 import { useWahlenStore } from "@/stores/wahlenStore.ts";
-import { useWahlvorstandStore } from "@/stores/wahlvorstandStore.ts";
-import { WahlWahlartEnum } from "@/types/wahl/WahlWahlartEnum.ts";
-import { WahlbezirksArtEnum } from "@/types/wahlbezirksArtEnum.ts";
 
 export function useTaskListService() {
-  const { initWahlen } = useWahlenStore();
-  const { initKonfigurationsparameter } = useInfomanagementStore();
-  const { initWahlvorstand } = useWahlvorstandStore();
-  const { initUngueltigeWahlscheine } = useWahlbezirkStore();
-  const { initKopfdaten } = useKopfdatenStore();
+  const { getWahlOrUndefinedById } = useWahlenStore();
+  const { currentUserWahlMetadata } = storeToRefs(useUserStore());
   const { currentUserWahlbezirksArt } = storeToRefs(useUserStore());
 
-  const matchingWahlbezirkArt = (task: Task) =>
-    task.onlyForWahlbezirksart === undefined ||
-    task.onlyForWahlbezirksart === currentUserWahlbezirksArt.value;
+  const { createTasks: createKopfdatenTasks } = useKopfdatenTaskFactory();
+  const { createTasks: createWahlvorstandTasks } = useWahlvorstandTaskFactory();
+  const { createTasks: createKonfigurationsparameterTasks } =
+    useKonfigurationsparameterTaskFactory();
+  const { createTasks: createUngueltigeWahlscheineTasks } =
+    useUngueltigeWahlscheineTaskFactory();
 
-  function getTaskList(): Task[] {
-    return _tasks.filter(matchingWahlbezirkArt);
+  function initTasklist() {
+    const tasks = [];
+    tasks.push(...initKopfdatenTaskList());
+    tasks.push(...initUngueltigeWahlscheineTaskList());
+    tasks.push(...initWahlvorstandTaskList());
+    tasks.push(...initKonfigurationsparameterTaskList());
+    return tasks;
   }
```

Comment:
Ein paar Import werden noch zu entfernen sein. Der andere Vorschlag in dem File gehört ist für diesen Vorschlag erforderlich.

```suggestion
  function initTasklist() {
    const taskFactoryData = _createTaskFactoryData();

    const tasks = [];
    tasks.push(...createKopfdatenTasks(taskFactoryData));
    tasks.push(...createUngueltigeWahlscheineTasks(taskFactoryData));
    tasks.push(...createWahlvorstandTasks(taskFactoryData));
    tasks.push(...createKonfigurationsparameterTasks(taskFactoryData));
    return tasks;
  }
```